.github/actions/spelling/expect.txt

```diff
@@ -384,3 +384,9 @@ XPLATSTR
 xsi
 Zanollo
 zy
+commande
```

Comment:
Several of these come from  `doc/windows/package-manager/package/manifest.md`:
```
ShortDescription: Le nouveau terminal Windows, une expérience de ligne de commande à onglets pour Windows.
```

There is a pattern for this line in `.github/actions/spelling/patterns.txt`:
```
# doc/ManifestSpecv1.0.md
^ShortDescription: "Le nouveau.*"$
```

Reply:
But the bot was still commenting the words as unrecognized.

Reply:
I think that's because the pattern doesn't match anymore without the quotes. If you remove the quotes from the pattern it would probably match the line again and we wouldn't need to list commande, une, rience, onglets:
```
^ShortDescription: Le nouveau.*$
```

Reply:
@lechacon It's failing again.